<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pixel Edit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollBar" name="horizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>768</y>
      <width>768</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="pageStep">
     <number>768</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>768</x>
      <y>0</y>
      <width>16</width>
      <height>768</height>
     </rect>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="pageStep">
     <number>768</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sel">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>384</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLabel" name="dispLbl">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>384</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>792</x>
      <y>272</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>792</x>
      <y>296</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>792</x>
      <y>320</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>792</x>
      <y>344</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Alpha</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_red">
    <property name="geometry">
     <rect>
      <x>862</x>
      <y>272</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_grn">
    <property name="geometry">
     <rect>
      <x>862</x>
      <y>296</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_blu">
    <property name="geometry">
     <rect>
      <x>862</x>
      <y>320</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_alp">
    <property name="geometry">
     <rect>
      <x>862</x>
      <y>344</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_opn">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>484</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sav">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>534</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_saa">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>584</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save As</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_lin">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>434</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Line Fill</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
